godot-core/src/registry.rs

```diff
@@ -200,19 +244,58 @@ pub fn auto_register_classes(init_level: InitLevel) {
         fill_class_info(elem.component.clone(), class_info);
     });
 
-    //out!("Class-map: {map:#?}");
+    let mut loaded_classes = get_loaded_classes_with_mutex();
+    let loaded_classes = loaded_classes.get_or_insert_with(Default::default);
```

Comment:
Maybe name `loaded_class_map` to differentiate from the further down used `loaded_classes` representing a vector.